### PR TITLE
chore(e2e): add healthchecks for flags announcements

### PR DESCRIPTION
**Pull request by mikhail-zlochevskyi**
#### What this PR does / why we need it:

Added healthchecks for:
/api/v0/settings/flags
/api/v0/announcements

[Why](https://www.neovasolutions.com/2022/09/29/comparison-of-api-request-libraries-axios-vs-supertest-vs-request/?utm_source=rss&utm_medium=rss&utm_campaign=comparison-of-api-request-libraries-axios-vs-supertest-vs-request) axios

#### Which issue(s) does this PR fixes?:

Fixes #

#### Sample Links & Screenshots:

Link:

<details>
<summary>Desktop Screenshot</summary>

</details>
<details>
<summary>Mobile Screenshot</summary>

</details>

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module.exports = function (api) {
```
Comment (thedoublejay) [inline babel.config.js:1]: We don't need this

Comment (thedoublejay) [reply babel.config.js:1]: data-test-id is only for cypress

File: package.json [hunk 2]
```diff
@@ -19,9 +21,12 @@
     "@waveshq/standard-web": "^0.33.0",
     "@waveshq/walletkit-core": "^0.27.0",
     "autoprefixer": "^10.4.13",
+    "axios": "^1.3.1",
+    "babel-plugin-jsx-remove-data-test-id": "^3.0.0",
```
Comment (thedoublejay) [inline package.json:25]: I think we can just add jest here

File: src/specs/healthcheck.spec.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+const baseUrl = "http://localhost:3000";
+
+const flags = "/api/v0/settings/flags";
+const announcements = "/api/v0/announcements";
+
+describe("API Healthcheck", () => {
+  it(`Should have status 200 for ${flags}`, () => {
+    axios
+      .get(baseUrl + flags)
```
Comment (thedoublejay) [inline src/specs/healthcheck.spec.ts:12]: Can we add a check here if the body is an array?

**Review by thedoublejay — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (mikhail-zlochevskyi) [reply package.json:25]: Sorry but using what we expect to perform an http GET request to the server?

**Pull request closed without merging**